.github/workflows/integrations.yml

```diff
@@ -162,6 +159,24 @@ jobs:
         run : |
           python -m pip install external/TraitGen
 
+      - name: Install TraitGen test dependencies
+        run : >
+          python -m pip install -r external/TraitGen/tests/requirements.txt
+          | awk '{print} /Attempting uninstall: openassetio/{found=1} END{exit found}'
```

Comment:
[Question] This seems like a very sensible thing to be testing. I wonder if there are other places we can keep this in mind.